dhis-2/dhis-api/pom.xml

```diff
@@ -35,10 +35,6 @@
       <groupId>net.sf.jasperreports</groupId>
       <artifactId>jasperreports</artifactId>
     </dependency>
-    <dependency>
-      <groupId>net.sf.jasperreports</groupId>
```

Comment:
I would really say yes if it is compiling and building and testing.

Reply:
That is not accurate though 😋 A module can still be built and tested even though its pom is not well specified. For example relying on transitive dependencies for dependencies that are actually direct ones.

Reply:
this one should be checked manually I think, we don't have tests for reports. Reports app -> standard reports use jasper.

Reply:
Yeah Ivo, you are right.
Reviewing them manually just javax-annotation is really used by dhis-api, all the others were here only because used by other modules that were importing dhis-api